opmon/network-performance-desktop-build-by-build.toml

```diff
@@ -0,0 +1,34 @@
+# Networking performance dashboard
+
+[project]
+
+name = "Build over build Network Performance for Desktop"
+platform = "firefox_desktop"
+xaxis = "build_id"
+start_date = "2023-01-01"
+metrics = [
+    "dns_lookup_time",
+    "http_page_tls_handshake",
+    "http_sub_tls_handshake",
+    "http_page_open_to_first_sent",
+    "time_to_response_start_ms"
+]
```

Comment:
These are defined here: [jetstream/outcomes/firefox_desktop/networking.toml](https://github.com/mozilla/metric-hub/blob/main/jetstream/outcomes/firefox_desktop/networking.toml)

Does that work?

Reply:
Unfortunately, no. Outcomes basically define a set of metrics that should always be computed together in Jetstream. Outcomes are currently not supported in OpMon (open issue https://github.com/mozilla/opmon/issues/153)

Generally, to reference metrics by their slugs, they'll need to be added as a definition in here: https://github.com/mozilla/metric-hub/tree/main/definitions

We are currently working on https://github.com/mozilla/jetstream/issues/1629 which will make it possible to reference these metric definitions in outcomes (which currently does not work). Once that is implemented you could move the metric definitions out of the outcome definition into a [definitions file](https://github.com/mozilla/metric-hub/blob/main/definitions/firefox_desktop.toml) and then simply reference the metrics in the outcome and in this config file. However, it'll probably take a week or two until we implement that support.

Until then, I'd suggest as a quick workaround to copy-paste the metric definitions into this config file (and then later simply delete them once they live in definitions).